Tasks/AzureResourceGroupDeployment/operations/ResourceGroup.ts

```diff
@@ -220,6 +220,7 @@ export class ResourceGroup {
         if (fileMatches.length < 1) {
             throw new Error(tl.loc("TemplateFilePatternMatchingNoFile"));
         }
+        console.log(tl.loc("DetectingOnlyUtf"));
```

Comment:
Lets not have this statement. I think the one in the previous file is good enough